lib/plugin.js

```diff
@@ -75,10 +75,32 @@ exports.register = function (server, opts, next) {
         patch: patch,
         delete: del
     })
-    
+
     server.ext('onPostStop', (server, next) => {
         adapter.disconnect(next)
     })
+
+
+    server.ext('onPreResponse', (request, reply) => {
+        const response = request.response
+        if (response.isBoom) {
+            response.output = response.output || {}
+            response.output.headers = response.output.headers || {}
+            response.output.headers['content-type'] = 'application/vnd.api+json'
```

Comment:
The content-type is not set anywhere else and it should be set to 'application/vnd.api+json' as per the JSON API spec.  This was the only way to get it to work. `reply.header(...)` isn't a function on this `reply` object.